express/parsers/apps/aiida/formats/zip.py

```diff
@@ -0,0 +1,84 @@
+import esse
+
+from express.parsers.formats.zip import BaseZipParser
+from express.parsers.formats.jsonfile import BaseJSONParser
+
+
+SUPPORTED_ARCHIVE_VERSION = '0.9'
+SUPPORTED_AIIDA_VERSION = '1.4.2'
+
+ES = esse.ESSE()
+SCHEMA_MATERIAL = ES.get_schema_by_id('material')
```

Comment:
Should we move these to `settings.py`? Perhaps in the `aiida` folder?

Reply:
I think these are fine.  I see, because ES is used in a static method below, it's not possible to define it as a property on `AiidaZipParser`.  Usually, these sort of global objects aren't desirable, but I think the scope is clean enough here that it doesn't hurt anything.  the `__all__` define here keeps it contained to this module.